logstash/templates/statefulset.yaml

```diff
@@ -0,0 +1,184 @@
+---
+apiVersion: {{ template "logstash.statefulset.apiVersion" . }}
+kind: StatefulSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: "{{ template "fullname" . }}"
+    chart: "{{ .Chart.Name }}"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    {{- range $key, $value := .Values.labels }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+spec:
+  selector:
+    matchLabels:
+      app: "{{ template "fullname" . }}"
+  replicas: {{ default .Values.replicas }}
```

Comment:
So this is in the Elasticsearch chart too so probably by fault. But I don't think this `default` actually does anything. It's always set by default and there is no other value for if its not set.